public/custom-assets/tingle.css

```diff
@@ -0,0 +1,267 @@
+/* ----------------------------------------------------------- */
```

Comment:
Why not put `tingle.min.js` & `tingle.css` in a `\tingle\` subfolder within `/custom-assets/`